dev/dags/example_virtualenv_mini.py

```diff
@@ -21,6 +26,8 @@
 )
 
 with DAG("example_virtualenv_mini", start_date=datetime(2022, 1, 1)) as dag:
+    pre = BashOperator(task_id="pre", bash_command="mkdir -p /tmp/persistent-venv2")
```

Comment:
I know this is still is draft, but taking an early review. I don't think this could be the right approach. Ideally `DbtSeedVirtualenvOperator` itself should create this path. Or is it creating a temporary at a different path? We may need to assert/use that, I guess

Reply:
also, what if we try `"/tmp/persistent-venv2"` instead of `Path("/tmp/persistent-venv2")` in the argument value for the operator? does that help?

Reply:
I think from a rendering point of view it should work, but type for virtualenv_dir is Path or None

Reply:
https://github.com/astronomer/astronomer-cosmos/blob/main/cosmos/operators/virtualenv.py#L79C9-L79C23

Reply:
I have created https://github.com/astronomer/astronomer-cosmos/pull/1721 with your suggestion if CI is happy we can close this PR and merge #1721

Reply:
yes, the CI was unhappy, but I pushed a commit https://github.com/astronomer/astronomer-cosmos/pull/1721/commits/1f1947c8c2fe3b3eb52a9c23aa04ae4c8dbd7572 that seems to help